unblob/handlers/compression/lzo.py

```diff
@@ -5,12 +5,27 @@
 
 from unblob.extractors import Command
 
+from ...file_utils import Endian, convert_int32
 from ...models import StructHandler, ValidChunk
 
 logger = get_logger()
 
+CHECKSUM_LENGTH = 4
 
-F_H_FILTER = 0x00000800
+F_ADLER32_D = 0x00000001  # noqa
+F_ADLER32_C = 0x00000002  # noqa
+F_STDIN = 0x00000004  # noqa
+F_STDOUT = 0x00000008  # noqa
+F_NAME_DEFAULT = 0x00000010  # noqa
+F_DOSISH = 0x00000020  # noqa
+F_H_EXTRA_FIELD = 0x00000040  # noqa
+F_H_GMTDIFF = 0x00000080  # noqa
+F_CRC32_D = 0x00000100  # noqa
+F_CRC32_C = 0x00000200  # noqa
+F_MULTIPART = 0x00000400  # noqa
+F_H_FILTER = 0x00000800  # noqa
+F_H_CRC32 = 0x00001000  # noqa
+F_H_PATH = 0x00002000  # noqa
```

Comment:
Do we really need the whole list of flags ? I'd keep the flags that we actually checks and remove the `noqa` comment.

Reply:
For future maintenance I would prefer either
- having all the structure & flag definitions
- a link to the spec

Reply:
Agree with @e3krisztian, I think it is better to list all flags for completeness. It makes understanding the file format easier.

There is unfortunately no specification of format, but added a comment on where to find the definition in the source.